typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -322,10 +327,14 @@ async function sendMessage(
   log('Intentionally setting interchain gas payment to 1');
 
   const channelStatsBefore = await app.channelStats(origin, destination);
-  const receipt = await utils.timeout(
-    app.sendHelloWorld(origin, destination, msg, value),
-    messageSendTimeout,
-    'Timeout sending message',
+  const receipt = await utils.retryAsync(
```

Comment:
This retry, sorry, was on mobile before

Reply:
There are errors that can't be retried at the provider level such as having the wrong nonce. I would be in favor of having this retry in here unless you think it is harmful

Reply:
I still don't think we should have it because the first attempt was to assess the state of the system and we should just report the error and move on and come back when the time comes. I think it is reasonable to say that if kathy (as it is right now) fails to send a message and then observe it on the foreign chain it means there is (or at least was) an issue that should be investigated. Checking again could give us an even more "WE REALLY NEED TO LOOK INTO" type message, but that does not seem to be the intent here?

Doing this will also risk delaying other tests the way it is written right now.